### PR TITLE
fea: qwen3next support

### PR DESCRIPTION
**Pull request by LiuXTao**
This PR provides the bridge for qwen3-next. It needs to be used in conjunction with the Mcore PR(https://github.com/NVIDIA/Megatron-LM/pull/1958) to ensure the correctness of delta_net out_norm. 

I have already aligned the forward logits between the hf model and mcore model.

### CODE REVIEW

**Comment by ISEEKYAN** [comment]
Thanks for your support, but this project is being deprecated, I sincerely recommend you to take a look at the [Megatron-Bridge](https://github.com/NVIDIA-NeMo/Megatron-Bridge) which is the official version with long term maintenance.
